SU-HSDO/suhumsci
# Hotfix: Update name for credit caption template

# READY FOR REVIEW

## Summary
We added a media--image.html.twig template to fix a WYSIWYG issue with media images.
This template has been overriding our caption/credit template.

This hotfix updates the caption/credit template naming to be correctly picked up in the template hierarchy.

Example prod site: https://iranian-studies.stanford.edu/
You will see the caption credit is floating outside of its image on the page:
![image](https://user-images.githubusercontent.com/9355698/128542568-559590c9-6275-4656-9161-72413f69fdf2.png)

This should be appearing on the lower image where there is a single spotlight view in a well:
![image](https://user-images.githubusercontent.com/9355698/128542632-e989b616-f6d7-4d11-a413-af6b8c95140b.png)

How it should look:
![image](https://user-images.githubusercontent.com/9355698/128542717-16dd7286-4cc3-4f46-b4a6-528b90b13a42.png)

This is being caused by the media template being picked up being wrong and not applying the position relative we have on caption credit media template, this work updates the naming of that template to correctly get picked up in image hierarchy.

## Need Review By (Date)
asap

## Urgency
🔥 

## Steps to Test
1. Clear your Drupal cache to pick up the new template naming
2. This work is one template rename, the best way to test it is to add images to a flexible page using caption/credits.
2. Check that single spotlight view paragraphs display the image correctly with the caption when logged out.
3. Create Text Area's with images in them, verify images still align correctly in the WYSIWYG when adding alignment.
4. Verify those same images added in Text Area display caption/credit correctly if a caption is added and saved on the page when logged out.
5. Create sliders, photo gallery grids, hero images, carousels and make sure that captions are not impacted by this template change, they should pick up the newly named template and appear correctly logged out.
5. Verify with inspector that the images that have captions are picking up the correct template: `media--image--caption-credit.html.twig`
![image](https://user-images.githubusercontent.com/9355698/128532482-1373864c-7f0e-4565-bef8-5478305be68f.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
